pkgs/applications/science/misc/root/default.nix

```diff
@@ -56,7 +56,7 @@
 
 stdenv.mkDerivation rec {
   pname = "root";
-  version = "6.32.04";
+  version = "6.32.06";
```

Comment:
```
error: hash mismatch in fixed-output derivation '/nix/store/k1apb5rg6953jpk3hc2xivcbfvvmkqrf-root_v6.32.06.source.tar.gz.drv':
         specified: sha256-Ey8Saq59MO+8zX3NmRt62hiQrleYDvMAwWQh+dTQfqg=
            got:    sha256-P8Ay2T/oSN6lrbG0fY8KhieVIyk/7gqis81Sof+rckc=
```
you forgot to update the hash for src

Reply:
Sorry, forgot to push the right hash before getting distracted by a meeting :)